seeders/20201109013159-create-demo-user.js

```diff
@@ -1,25 +1,53 @@
 /* eslint-disable */
+const {ROLE_ADMIN, ROLE_USER} = require('../constants/user.constants')
+const password = require('../helpers/password')
+
+const NUMBER_OF_ADMIN_USERS = 10
+const NUMBER_OF_REGULAR_USERS = 10
+
+async function createDemoUsers (numberOfAdmin, numberOfUser) {
+  let users = []
+  for (let i=1; i <= numberOfAdmin; i++) {
+    users.push({
+        firstName: `User ${i}`,
+        lastName: 'Admin',
+        email: `admin${i}@test.com`,
+        // Important: Password not encrypted yet!
+        password: await password.encrypt(`admin${i}`),
+        roleId: ROLE_ADMIN,
+        image: 'https://www.designevo.com/res/templates/thumb_small/colorful-hand-and-warm-community.png',
+        createdAt: new Date(),
+        updatedAt: new Date(),
+    })
+  }
+  for (let i=1; i <= numberOfUser; i++) {
+    users.push({
+        firstName: `User ${i}`,
+        lastName: 'Regular',
+        email: `user${i}@test.com`,
+        // Important: Password not encrypted yet!
+        password: await password.encrypt(`user${i}`),
+        roleId: ROLE_USER,
+        image: 'https://www.designevo.com/res/templates/thumb_small/colorful-hand-and-warm-community.png',
+        createdAt: new Date(),
+        updatedAt: new Date(),
+    })
+  }
```

Comment:
Pensar la función de manera que la misma pueda ser reutilizada para crear distintos tipos de usuarios, si tenes 10 tipos de roles, tendrias una aridad de 10 argumentos y 10 bucles for dentro de la funcion.

Reply:
Ahora le paso a la función un array con los distintos tipos de roles/usuarios, por lo que la función que crea los usuarios ahora es independiente del los roles existentes o que puedan existir en el futuro.